### PR TITLE
[Chore]: Technical: Translate geojson-layer

### PR DESCRIPTION
**Pull request by HeimEndyd**

### CODE REVIEW

File: src/layers/geojson-layer/geojson-layer.ts [hunk 1]
```diff
@@ -60,15 +70,70 @@ export const geojsonVisConfigs = {
...
+  heightRange: [number, number];
+  elevationScale: number;
+  enableElevationZoomFactor: boolean;
+  stroked: boolean;
+  filled: boolean;
+  enable3d: boolean;
+  wireframe: boolean;
+};
+
+export type GeoJsonLayerConfig = Merge<
```
Comment (heshan0131) [inline src/layers/geojson-layer/geojson-layer.ts:117]:
```js
type GeoJsonLayerVisualChannelConfig = LayerColorConfig & LayerStrokeColorConfig 
LayerSizeConfig & LayerHeightConfig & LayerRadiusConfig
```

File: src/layers/geojson-layer/geojson-utils.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ import wktParser from 'wellknown';
 import normalize from '@mapbox/geojson-normalize';
```
Comment (heshan0131) [inline src/layers/geojson-layer/geojson-utils.ts:22]: Please types below to type the functions in this file
```js
import {KeplerTable} from '../../utils';
import {Feature, BBox} from 'geojson';
import {LayerColumns} from '../index';

type GetFeature = (d: any) => Feature;
type GeojsonDataMaps = Array<Feature | null>;

export enum FeatureTypes {
  Point = 'Point',
  MultiPoint = 'MultiPoint',
  LineString = 'LineString',
  MultiLineString = 'MultiLineString',
  Polygon = 'Polygon',
  MultiPolygon = 'MultiPolygon'
}

type FeatureTypeMap = {
  [key in FeatureTypes]: boolean;
};
export function parseGeoJsonRawFeature(rawFeature: unknown): Feature | null;

export function getGeojsonDataMaps(
  allData: KeplerTable['allData'],
  getFeature: GetFeature
): GeojsonDataMaps;

export function getGeojsonBounds(features: Feature[]): BBox | null;

export function getGeojsonFeatureTypes(allFeatures: Feature[]): FeatureTypeMap;

export function parseGeometryFromString(geoString: string): Feature | null;

```

File: src/layers/geojson-layer/geojson-layer.ts [hunk 3]
```diff
@@ -60,15 +70,70 @@ export const geojsonVisConfigs = {
...
 
 // access feature properties from geojson sub layer
 export const defaultElevation = 500;
 export const defaultLineWidth = 1;
 export const defaultRadius = 1;
 
 export default class GeoJsonLayer extends Layer {
+  declare config: GeoJsonLayerConfig;
+  declare visConfigSettings: GeoJsonVisConfigSettings;
+  dataToFeature: {properties?: {radius?: number}}[];
```
Comment (heshan0131) [inline src/layers/geojson-layer/geojson-layer.ts:135]: `dataToFeature: GeojsonDataMaps`

File: src/layers/geojson-layer/geojson-layer.ts (hunk 1 above)
Comment (HeimEndyd) [reply src/layers/geojson-layer/geojson-layer.ts:117]: What is meant by LayerRadiusConfig?

Comment (heshan0131) [reply src/layers/geojson-layer/geojson-layer.ts:135]: You can add this to base-layer
```js
export type LayerRadiusConfig = {
  radiusField: VisualChannelField;
  radiusDomain: VisualChannelDomain;
  radiusScale: VisualChannelScale;
};

```

**Pull request merged**